### PR TITLE
feat(console): user connector delete confirmation

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat: User Details Page Connectors Deletion Confirm

## Linear Issue Reference

N / A

## Testing

image

### CODE REVIEW

File: packages/console/src/pages/UserDetails/components/UserConnectors.tsx [hunk 1]
```diff
@@ -127,7 +129,8 @@ const UserConnectors = ({ userId, connectors, onDelete }: Props) => {
                     title="admin_console.user_details.connectors.remove"
                     type="plain"
                     onClick={() => {
-                      void handleDelete(id);
+                      const connector = displayConnectors.find((connector) => connector.id === id);
```
Comment (simeng-li) [inline packages/console/src/pages/UserDetails/components/UserConnectors.tsx:132]: Looks like no need to loop again to find the connector here.  Just simply declare the connector in the map method. `displayConnectors.map((connetor) => {})`

Comment (IceHe) [reply packages/console/src/pages/UserDetails/components/UserConnectors.tsx:132]: e.g.?
```ts
const hasConnector = displayConnectors.some((connector) => connector.id === id);
```

File: packages/phrases/src/locales/zh-cn.ts [hunk 2]
```diff
@@ -363,6 +363,7 @@ const translation = {
         user_id: '用户ID',
         remove: '删除',
         not_connected: '该用户还没有绑定社交账号。',
+        deletion_confirmation: '您在正要删除现有的 <name /> 身份，是否确认？',
```
[the remark this answers is not in the document]
Comment (IceHe) [reply packages/phrases/src/locales/zh-cn.ts:366]: just option `您正在删除 <name /> 的身份信息，是否确定要这样做？`?

File: packages/console/src/pages/UserDetails/components/UserConnectors.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/UserDetails/components/UserConnectors.tsx]: updated.

File: packages/phrases/src/locales/en.ts [hunk 3]
```diff
@@ -372,6 +372,8 @@ const translation = {
         user_id: 'User ID',
         remove: 'Remove',
         not_connected: 'The user is not connected to any social connector.',
+        deletion_confirmation:
+          'You are removing the existing <name/> identity. Are you want to do that?',
```
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/phrases/src/locales/en.ts:376]: This is not related to this PR.
Refer to: https://silverhand-io.slack.com/archives/C0243P3E4KS/p1655784066250849

File: packages/phrases/src/locales/zh-cn.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/phrases/src/locales/zh-cn.ts:366]: This is not related to this PR.
Refer to: https://silverhand-io.slack.com/archives/C0243P3E4KS/p1655784066250849

**Review by wangsijie — APPROVED** [review]

**Pull request merged**